lib/ruby_gems_download_strategy.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'download_strategy'
+
+class RubyGemsDownloadStrategy < AbstractDownloadStrategy
+  class << self
+    def gem_config
+      {
+        backtrace: false,
+        bulk_threshold: 1000,
+        sources: [
+          "https://x-oauth-basic:#{ENV['HOMEBREW_GITHUB_API_TOKEN']}@rubygems.pkg.github.com/bodyshopbidsdotcom/",
+          'http://rubygems.org/'
+        ],
+        update_sources: true,
+        verbose: true
+      }
+    end
+
+    def gem_config_file(&block)
+      file_path = File.join(Dir.tmpdir, '.gemrc')
+
+      open(file_path, 'w') { |file| file.write(YAML.dump(gem_config)) }
+      block.call(file_path)
+    ensure
+      File.unlink(file_path)
+    end
+  end
+
+  def cached_location
+    Pathname.new("#{HOMEBREW_CACHE}/#{name}-#{version}.gem")
+  end
+
+  def clear_cache
+    cached_location.unlink if cached_location.exist?
+  end
+
+  def setup_debug_tools
+    Homebrew.install_gem_setup_path! 'pry'
+    Homebrew.install_gem_setup_path! 'pry-byebug', executable: 'pry'
+    Homebrew.install_gem_setup_path! 'dotenv'
+    # require 'dotenv/load'
```

Comment:
If we're not using dotenv, we should remove it from here and from the counterpart in the Tinker formula.

Reply:
Removed references to `dotenv`